ui/src/app/login/components/login.tsx

```diff
@@ -5,6 +5,7 @@ import {uiUrl, uiUrlWithParams} from '../../shared/base';
 import {useCollectEvent} from '../../shared/components/use-collect-event';
 
 import './login.scss';
+import {useTranslation} from 'react-i18next';
```

Comment:
Please follow the existing conventions within these files -- this import should be at the top of this file, where all the external imports are